plugin-infra/go-plugin-access/src/test/java/com/thoughtworks/go/plugin/access/notification/v3/JsonMessageHandler3_0_Test.java

```diff
@@ -330,7 +331,7 @@ private String errorMessageForNotifyResult(String message) {
     }
 
     private Date getFixedDate() throws Exception {
-        return new SimpleDateFormat(DATE_PATTERN).parse("2011-07-13T19:43:37.100Z");
+        return new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ").parse("2011-07-13T19:43:37.100+0530");
```

Comment:
Can we use calendar to build date object -

```
Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"));
        calendar.set(2011,07,13,19,43,37);
        calendar.set(Calendar.MILLISECOND,100);
        return calendar.getTime();
```

If we take above approach then we can remove https://github.com/gocd/gocd/pull/4360/files#diff-21f722c49e082ef198454f08e361be18R358 and https://github.com/gocd/gocd/pull/4360/files#diff-21f722c49e082ef198454f08e361be18R104 from test since we have a fixed date in `createPipeline ()`